src/power_grid_model/validation/__init__.py

```diff
@@ -4,11 +4,14 @@
 
 """Power Grid Model input/update data validation"""
 
-from .assertions import (
+from power_grid_model.validation.assertions import (
     ValidationException,
     assert_valid_batch_data,
     assert_valid_input_data,
 )
-from .errors import ValidationError
-from .utils import InputData, UpdateData, errors_to_string
-from .validation import validate_batch_data, validate_input_data
+from power_grid_model.validation.errors import ValidationError
```

Comment:
So absolute import does work for `__init__.py`?

Shall we then also change the base `__init__.py` to absolute import?

Reply:
Yes, I'll try. I remember that there was some issue with the base __init__.py, but I don't remember what it was :)